src/pages/pull/read/components/Sidebar.tsx

```diff
@@ -63,7 +64,7 @@ export default function Sidebar() {
               ) : (
                 <PiDotDuotone className="min-w-[24px] h-6 !text-yellow-500" />
               )}
-              <span className="truncate">{reviewer.address}</span>
+              <span className="truncate">{shortenAddress(reviewer.address, 6)}</span>
```

Comment:
Does it have a way to see the full address? Like a tooltip when you hover for a few seconds?